code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/TyagunovVladimirTest.java

```diff
@@ -0,0 +1,35 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
@VladimirTyagunov , please, remove the star import

Reply:
Fixed.